cmd/example/helloworld/main.go

```diff
@@ -66,22 +66,30 @@ func newCommand() *cobra.Command {
 }
 
 func newControllerCommand() *cobra.Command {
+	o := addonmanager.NewManagerOptions()
```

Comment:
we should not impact the existing addon. We could build a new addon example that supports cloudevent, or select one addon as an example. It also ensure that existing addon won't be impacted by this change.

Reply:
built new example that supports cloudevent, following the existing examples.